integ-test/src/test/resources/correctness/queries/flowcontrol.txt

```diff
@@ -0,0 +1,8 @@
+SELECT IFNULL(AvgTicketPrice, AvgTicketPrice * 2.0) FROM kibana_sample_data_flights
```

Comment:
To simplify, you can move these tests to https://github.com/opendistro-for-elasticsearch/sql/blob/develop/integ-test/src/test/resources/correctness/expressions/conditionals.txt. In this way, you don't need a `FROM` clause and test index. The only literal not supported is `NULL` but you've figured out the workaround (1/0).

Reply:
done.